safety/safety.py

```diff
@@ -84,6 +84,10 @@ def fetch_database_url(mirror, db_name, key, cached, proxy):
         return data
     elif r.status_code == 403:
         raise InvalidKeyError()
+    else:
+        cached_data = get_from_cache(db_name=db_name, site_down=True)
```

Comment:
This is a good thing for other HTTP status codes, but we still need to catch **requests** connection exceptions as well.